starter-code/iteration-2/main.ts

```diff
@@ -1,6 +1,32 @@
 // Add the reference to the "TodoInterface"
-
+import {TodoInterface} from './todoInterface'
 // 1. Create a class Todo that implements the Interface created before.
+class Todo implements TodoInterface{
+  list: Array<string> = [];
+  addTask(nameTask:string):number{
+    this.list.push(nameTask);
+    console.log("Task: " + nameTask + " inserted in the list");
+    return this.list.length;
+  }
+
+
+  // adasvsdfsdfsd
+  listAllTasks():void{
+    for(var i =0; i<this.list.length; i++){
+      console.log(this.list[i]);
+    }
+  }
+  deleteTask(nameTask:string):number{
```

Comment:
you could use a splice here as well:
`deleteTask(task: string): number {
    var index = this.todoList.indexOf(task);
    this.todoList.splice(index, 1);
    console.log(`Task "${task}" removed from the list`);
    return this.todoList.length;
  }
}`